src/components/navbar/index.css

```diff
@@ -4,4 +4,12 @@
     min-height: 7vh;
     padding: 1.5rem 3rem;
     font-size: 1.4rem;
+}
+
+.auth_btn {
+    color: var(--black) !important;
```

Comment:
what is the purpose of using !important here ?

Reply:
To override link styles.